pkg/controller/ps/controller.go

```diff
@@ -513,11 +517,21 @@ func (r *PerconaServerMySQLReconciler) reconcileMySQLAutoConfig(ctx context.Cont
 
 		return nil
 	}
+
+	config := ""
+
+	// for single-node clusters, we need to set read_only=0 if rchestrator is disabled
+	if cr.MySQLSpec().Size == 1 && !cr.Spec.Orchestrator.Enabled {
```

Comment:
if memory resource requirements are not set, will it work?

Reply:
on line 502 we check `if memory == nil` and early return if true

Reply:
Oh, yes, you're right.

Reply:
[da2140e](https://github.com/percona/percona-server-mysql-operator/pull/562/commits/da2140e687e2f2d03f16894b4f2091d554f86132)